README.md

```diff
@@ -56,6 +56,31 @@ Inside the checked out repository, running `cargo build --release` will build `a
 
 From within the repository, you can also run, e.g., `cargo install --locked --path . --root /usr/local`, which will copy `agora` to `/usr/local/bin/agora`.
 
+## Running with Docker
+
+The `agora` Docker image can be pulled from `TBD`. 
+
+### Building Agora Docker Image
+
+The Docker image can also be built directly from within the repository. The Dockerfile will copy the `/files` and `/.lnd`directory into the container and serve all files in the `/files` directory and search the `.lnd` directory for the `tls.cert` and `invoices.macaroon` files. The `.lnd` directory is only needed if you plan on connecting an LND instance.
```

Comment:
Again, you probably don't want to copy the `.lnd` directory into the docker image that you build. The idea of a docker image is to build it once, and make it re-usable. That won't be possible if you have a specific `.lnd` baked into the image.